pkg/cmos/cmos.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2012-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package cmos lets you read and write to cmos registers while doing basic checks on valid register selections.
+
+package cmos
+
+import (
+	"fmt"
+	"github.com/u-root/u-root/pkg/memio"
+)
+
+const (
+	CMOS_REG_PORT  = 0x70
```

Comment:
Go doesn't usually like constants like these. We tolerate them where they reflect some other code base and are really commonly used (Linux kernel constants, etc). I think in this case I'd prefer camelcase https://golang.org/doc/effective_go.html#mixed-caps

Reply:
I guess this is my fault from the other PR